test/browser/browser.sh

```diff
@@ -42,11 +42,9 @@ fi
 if grep -q 'ID=.*fedora' /etc/os-release && [ "$PLAN" = "storage-basic" ]; then
     # required by TestStorageBtrfs*
     dnf install -y udisks2-btrfs
-    # required by TestStorageStratis
-    dnf install -y stratisd-tools
 fi
 
-if grep -q 'ID=.*centos' /etc/os-release && grep -q 'VERSION_ID=.*10' /etc/os-release && [ "$PLAN" = "storage-basic" ]; then
+if grep -q 'ID_LIKE=.*fedora' /etc/os-release && [ "$PLAN" = "storage-basic" ]; then
```

Comment:
That is a no-op, as it's true everywhere. If that works, please just add it to test/browser/main.fmf next to stratis-cli.

Reply:
Actually, Fedora does *not* have `ID_LIKE=fedora`. So it will not actually install it there, bu the commit message says otherwise?